book/src/snapshot-verification.md

```diff
@@ -0,0 +1,34 @@
+# Verifying Snapshots and Account state proof
+
+A snapshot includes the account state for each live account in the system at a given slot in a bundle of append vecs.
+
+To verify the snapshot, we propose the following:
+
+On account store hash the following data:
+* Account owner
+* Account data
+* Account pubkey
+* Account lamports balance
+* Fork the account is stored on
```

Comment:
Do we snapshot forks? Not just rooted accounts?

Reply:
Just rooted accounts. This is to avoid creating the same hash value for an identical account state on fork A and fork B.

Reply:
The fork information is in the snapshot as well so one would know what fork a given account is stored on.